lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/AvroClassHierarchy.cs

```diff
@@ -206,6 +236,33 @@ public INode GetNode(string fullName)
             return ret;
         }
 
+
+        public INode GetNode(string fullName, string aliasLanguage)
```

Comment:
Actually, lets do all of that as a separate JIRA. I will review, test and merge this.